apps/rush-lib/src/logic/deploy/DeployManager.ts

```diff
@@ -461,9 +473,16 @@ export class DeployManager {
     const deployMetadataJson: IDeployMetadataJson = {
       scenarioName: path.basename(deployState.scenarioFilePath),
       mainProjectName: deployState.mainProjectName,
+      projects: [],
       links: []
     };
 
+    deployState.folderInfosByPath.forEach((folderInfo) => {
```

Comment:
This is including too many projects. It should include only the ones where [IDeployState.folderInfosByPath](https://github.com/microsoft/rushstack/blob/89076aee15970ccab968c6ca48b85fc7165359df/apps/rush-lib/src/logic/deploy/DeployManager.ts#L99) has `isRushProject=true`.